### ISSUE TITLE
WANTED: ability to supress version warning

### ISSUE THREAD

**Issue by travisdowns** [microsoft/pyright#11022]
I would like a way to suppress the version warning message:

```
WARNING: there is a new pyright version available (v1.1.405 -> v1.1.406).
```

I'm down for updating pyright now and then to keep it up to date but the rate of releases is high and the way we run pyright (for subsets of files) causes a bunch of these messages to clutter our logs.

**Comment by erictraut** [comment]
Thanks for the request. This warning isn't coming from pyright itself. It's coming from the community-maintained [Python wrapper](https://github.com/RobertCraigie/pyright-python) for pyright. Please post your feature request in that project's issue tracker.

**Comment by travisdowns** [comment]
> Thanks for the request. This warning isn't coming from pyright itself. It's coming from the community-maintained [Python wrapper](https://github.com/RobertCraigie/pyright-python) for pyright. Please post your feature request in that project's issue tracker.

Thanks! It also explains why I couldn't find the relevant strings in the typescript source :).
